### PR TITLE
Remove parameters from mapd Fixes #1610

### LINKED ISSUE #1610: Error : Backend=MapD ,  client.create_table(partition=['yyyy', 'mm'])  error!!
Backend=MapD  Python=3.6  MapD=4.1  ibis=0.14

import ibis

client = ibis.mapd.connect(host='127.0.0.1', user='user_test', password='123456',port=9091, database='db_test')

struct = ibis.schema([('txt', 'string'),
                                 ('yyyy', 'int32'),
                                 ('mm', 'int16')])
                      
client.create_table('table_0', schema=struct, partition=['yyyy', 'mm']) 
Return : TypeError: create_table() got an unexpected keyword argument 'partition'

### PR DESCRIPTION
**Pull request by tonyfast**
Closes #1610

### CODE REVIEW

**Comment by cpcloud** [comment]
@tonyfast Can you add a test that hits this code?

**Comment by tonyfast** [comment]
@cpcloud I tried to add some tests.  I am receiving some errors that I don't completely understand.  Do I need to add some skip expressions somewhere?

**Comment by cpcloud** [comment]
@tonyfast Can you kick off the build again?

File: ibis/mapd/tests/conftest.py [hunk 1]
```diff
@@ -50,3 +51,33 @@ def translate():
...
+
+
+@pytest.fixture
+def temp_table(con):
+    name = _random_identifier('table')
+    try:
+        yield name
+    finally:
+        assert con.exists_table(name), name
+        con.drop_table(name)
```
Comment (xmnlab) [inline ibis/mapd/tests/conftest.py]: @tonyfast not sure why but in the past I had some problems with drop function that broke the mapd server on CI

**Comment by cpcloud** [comment]
@tonyfast Please rebase on master, and you should have a green build assuming your tests are passing.

**Comment by tonyfast** [comment]
Thanks for the fixes @cpcloud . It looks like things are passing now.

**Comment by cpcloud** [comment]
@tonyfast Does this have an associated issue that should closed?

**Comment by tonyfast** [comment]
Closes #1610

**Pull request closed without merging**